app/src/main/java/com/github/drawyourpath/bootcamp/FirebaseBootcampActivity.kt

```diff
@@ -0,0 +1,32 @@
+package com.github.drawyourpath.bootcamp
+
+import android.os.Bundle
+import android.widget.Button
+import android.widget.EditText
+import androidx.appcompat.app.AppCompatActivity
+
+
+class FirebaseBootcampActivity : AppCompatActivity() {
+
+    var dataBase: Database = FireDatabase()
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_firebase_bootcamp)
+
+        //test if we are running tests in the dataBase
+        val isTest: Boolean = intent.extras?.getBoolean("isRunningTestForDataBase") ?: false
```

Comment:
I don't think we could replace by this line because if we aren't in the test scenario we don't have an intent extra so we must have a default value to no use the MockDataBase which is here the value false.

Reply:
ok, sorry. I hadn't understand that the second argument was the default value. Thank you for the tips